packages/plugin-cloudflare/src/tunnel.ts

```diff
@@ -45,7 +46,13 @@ class TunnelClientInstance implements TunnelClient {
 
   constructor(port: number) {
     this.port = port
-    this.tunnel()
+    install()
+      .then(() => {
+        this.tunnel()
+      })
+      .catch((error) => {
```

Comment:
I think it'd be good to have a test case for the error scenario too, to ensure that code path is exercised in test.